Packages/Core/Editor/Generator/ClassModifier.cs

```diff
@@ -0,0 +1,9 @@
+namespace UnityAtoms.Editor
+{
+    public enum ClassModifier
```

Comment:
Virtuals, Partials, how many need to be here?

Reply:
Partial would be a different kind of modifier, since a class can be "partial sealed" or "partial abstract".

Reply:
Keep in mind that "public, private, protected" and similar would be an access modifier. I didn't write this whole generator system out as much since we don't need all that for Atoms and since actually writing out such a system would take a lot of work getting the details right.